docs/guide/fields.md

```diff
@@ -116,6 +116,51 @@ strawberry_django.field_type_map.update({
 })
 ```
 
+## Including / excluding Django model fields by name
+
+`strawberry.django.type` includes two optional keyword fields to help you more easily populate fields from the Django model, `fields` and `exclude`.
+
+Valid values for `fields` are:
+
+- `__all__` to assign `strawberry.auto` as the field type for all model fields.
+- `[<List of field names>]` to assign `strawberry.auto` as the field type for the enumerated fields. These can be combined with manual type annotations if needed.
+
+```{.python title=fields_all.py}
```

Comment:
In this case, since all those would not actually go to difference files, maybe define the title to something more semantic?

Like `title="All Fields"`, and `title="Enumerated Fields"` below, and so on